data/assets/examples/renderable/renderablesphereimageonline/simplesphereimageonline.asset

```diff
@@ -0,0 +1,28 @@
+local Radius = 696340000 * 20 -- The radius of the Sun in meters, times 20
+
+local SimpleSphereOnlineImage = {
```

Comment:
One suggestion was to use the `Node` as the name for individual scene graph nodes so that that part looks the same for all?  Thoughts